blueocean-pipeline-editor/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Do we have anything in place yet that will help us keep these files the same?

Reply:
No.  I tried to symlink them but tooling seems to not like that.